VideoWeb/VideoWeb/ClientApp/src/app/services/launch-darkly.service.ts

```diff
@@ -0,0 +1,54 @@
+import { Injectable } from '@angular/core';
+import * as LDClient from 'launchdarkly-js-client-sdk';
+import { ReplaySubject } from 'rxjs';
+import { ConfigService } from './api/config.service';
+
+export const FEATURE_FLAGS = {
+    vhoWorkAllocation: 'vho-work-allocation'
+};
+
+@Injectable({
+    providedIn: 'root'
+})
+export class LaunchDarklyService {
+    private flags: any;
+    ldClient: LDClient.LDClient;
+    flagChange = new ReplaySubject();
+
+    constructor(private configService: ConfigService) {
+        this.initialize();
+
+        this.onReady();
+
+        this.onChange();
+    }
+
+    initialize(): void {
+        this.flags = {};
+        const ldClientId = this.configService.getConfig().launch_darkly_client_id;
+        const user: LDClient.LDUser = { key: 'VideoWeb', anonymous: true };
+        this.ldClient = LDClient.initialize(ldClientId, user);
+    }
+
+    onReady(): void {
+        this.ldClient.on('ready', flags => {
+            this.setAllFlags();
+        });
+    }
+
+    onChange(): void {
+        this.ldClient.on('change', flags => {
+            for (const flag of Object.keys(flags)) {
+                this.flags[flag] = flags[flag].current;
+            }
+            this.flagChange.next(this.flags);
+            console.log('Flags updated', this.flags);
```

Comment:
is there anything stopping us from using the logger service instead of console.log reaching outside of dev

Reply:
I dont think so. An oversight by me actually, ill update it.